tests/integration/examples/custom_formatters/load_formatter_failed.py

```diff
@@ -0,0 +1,6 @@
+from djl_python.input_parser import input_formatter
+from djl_python.output_formatter import output_formatter
+
+@input_formatter
+def custom_input_formatter(decoded_payload, **kwargs):
+    invalid syntax here to cause load error
```

Comment:
This is causing continuous build failed:

```
> Task :verifyPython
yapf: ./tests/integration/examples/custom_formatters/load_formatter_failed.py:6:13: invalid syntax
```

Reply:
Is it possible to do something like this? Otherwise the continuous ci will keep in failed state.

```
def custom_input_formatter(decoded_payload, **kwargs):
    raise ValueError("invalid syntax here to cause load error")
```